mythril/laser/ethereum/cfg.py

```diff
@@ -83,6 +86,30 @@ def get_cfg_dict(self) -> Dict:
             code=code,
         )
 
+    def merge_nodes(self, node: "Node", constraints: Constraints):
+        """
+        Merges two nodes
+        :param node: The node to be merged
+        :param constraints: The merged constraints
+        :return:
+        """
+        self.states += node.states
+        self.uid = get_new_gbl_id()
+        self.flags |= node.flags
+        self.constraints = constraints
+
+    def check_merge_condition(self, node: "Node"):
```

Comment:
Can the business logic related to state merging be confined to the plugin? Or is it really necessry to add these helpers here?